packages/gui/src/hooks/useOfferExpirationDefaultTime.tsx

```diff
@@ -47,15 +47,13 @@ function stringPropertiesToNumbers(objWithStrings: OfferExpirationDefaultTimeInp
 }
 
 export default function useOfferExpirationDefaultTime() {
+  const defaultsCopy = { ...offerExpirationDefaultTimeDefaults };
```

Comment:
We were returning the `offerExpirationDefaultTimeDefaults` object, allowing direct write access to the object. This caused issues with the logic for updating the persisted pref values.